extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -23,45 +23,30 @@ protected function runOnPageSet( ApiPageSet $pageSet ) {
 
 			if ( is_array( $parsedInfoboxes ) && count( $parsedInfoboxes ) ) {
 				$inf = [ ];
+
 				foreach ( array_keys( $parsedInfoboxes ) as $k => $v ) {
 					$inf[ $k ] = [ ];
 				}
+
 				$pageSet->getResult()->setIndexedTagName( $inf, 'infobox' );
 				$pageSet->getResult()->addValue( [ 'query', 'pages', $id ], 'infoboxes', $inf );
-				foreach ( $parsedInfoboxes as $count => $infobox ) {
-					$sl = isset( $infobox[ 'sourcelabels' ] ) ?
-						$infobox[ 'sourcelabels' ] :
-						$this->sourceLabelsFallback( $infobox, $articleTitle );
 
+				foreach ( $parsedInfoboxes as $count => $infobox ) {
 					$pageSet->getResult()->addValue( [ 'query', 'pages', $id, 'infoboxes', $count ], 'id', $count );
-					$pageSet->getResult()->setIndexedTagName( $sl, "sourcelabels" );
+
 					$pageSet->getResult()->addValue(
-						[ 'query', 'pages', $id, 'infoboxes', $count ], 'sourcelabels', $sl
+						[ 'query', 'pages', $id, 'infoboxes', $count ],
+						'parser_tag_version',
+						$infobox['parser_tag_version']
+					);
+
+					// FIXME ?format=xml throws error
+					$pageSet->getResult()->setIndexedTagName( $infobox['metadata'], 'metadata' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "metadata" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)